src/app/wizard-extension/organisation-wizard-extension/organisation-wizard-extension.component.html

```diff
@@ -256,42 +248,34 @@
                         <mat-panel-title>
                             <div>
                                 Signatur-Einstellungen
-                                <div *ngIf="!isSignerConfigValid()" style="color: red">
-                                    <small>
-                                        Bitte überprüfen Sie die Signatur-Einstellungen.
-                                    </small>
-                                </div>
                             </div>
                         </mat-panel-title>
                     </mat-expansion-panel-header>
 
-                    <div class="mb-3">
+                    <div hidden class="mb-3">
                         <label class="form-label">Private Key</label>
-                        <textarea rows="10" class="form-control" [(ngModel)]="orgaMetadata.organisationSignerConfigDto.privateKey">
+                        <textarea rows="10" class="form-control" [value]="orgaMetadata.organisationSignerConfigDto.privateKey" [disabled]="true">
                         </textarea>
                         <div cFormText style="color: #7f7f7f;">
-                            <small>Der Private Key zur unten genannten Verifizierungsmethode. Wird zum Signieren von Selbstbeschreibungen verwendet.</small>
-                        </div>
-                        <div *ngIf="!isFieldFilled(orgaMetadata.organisationSignerConfigDto.privateKey)" style="color: red">
-                            <small>
-                                Private Key ist erforderlich.
-                            </small>
+                            <small>Der Private Key zur unten genannten Verifizierungsmethode. Kann zum Signieren von Selbstbeschreibungen verwendet werden.</small>
```

Comment:
we should hide the private key and verification method as they are no longer used and only confuse users